sopel/plugin.py

```diff
@@ -896,6 +897,44 @@ def guarded(bot, trigger, *args, **kwargs):
     return actual_decorator
 
 
+def require_bot_privilege(level, message=None, reply=False):
+    """Decorate a function to require the bot a privilege level or above.
+
+    :param int level: required privilege level to use this command
+    :param str message: optional message said to insufficiently privileged user
```

Comment:
I wanted to correct "insufficiently privileged user" because I know exactly where that was copied-and-pasted from 😝 and I just kept going after that.

```suggestion
    :param int level: minimum channel privilege the bot needs for this function
    :param str message: optional message said if the bot's channel privilege
                        level is insufficient
```